pkg/scanners/trivy/trivy.go

```diff
@@ -240,5 +247,45 @@ func main() {
 		os.Exit(generalErr)
 	}
 
+	// configure scanner metrics
+	ctxB := context.Background()
+	ctx, cancel := signal.NotifyContext(ctxB, os.Interrupt, syscall.SIGTERM)
+	defer cancel()
+
+	exporter, err := otlpmetrichttp.New(ctx, otlpmetrichttp.WithInsecure(), otlpmetrichttp.WithEndpoint(os.Getenv("OTEL_EXPORTER_OTLP_ENDPOINT")))
```

Comment:
i think you mentioned we need an otel collector to receive the metrics, can we add docs on how to set that up?